xbmc/platform/darwin/tvos/input/LibInputSettings.h

```diff
@@ -10,8 +10,8 @@
 
 @interface TVOSLibInputSettings : NSObject
 
-@property(nonatomic) bool useSiriRemote;
-@property(nonatomic) BOOL remoteIdleEnabled;
-@property(nonatomic) int remoteIdleTimeout;
+@property(nonatomic) bool siriRemotePanEnabled;
+@property(nonatomic) bool siriRemoteIdleTimer;
```

Comment:
better name it `isSiriRemoteIdleTimer` or `siriRemoteIdleTimerEnabled` or something similar